u2f-api.js

```diff
@@ -10,289 +10,291 @@
 
 'use strict';
 
-/** Namespace for the U2F api.
- * @type {Object}
- */
-var u2f = u2f || {};
+if (!('u2f' in window) && 'chrome' in window && chrome.runtime) {
+  /** Namespace for the U2F api.
+   * @type {Object}
+   */
+  var u2f = {};
```

Comment:
@dgraham Would it make sense that this line is getting hoisted, causing `'u2f' in window` to return `true`?